api/envoy/extensions/transport_sockets/tls/v4alpha/tls.proto

```diff
@@ -96,11 +97,23 @@ message DownstreamTlsContext {
 }
 
 // TLS context shared by both client and server TLS contexts.
-// [#next-free-field: 9]
+// [#next-free-field: 11]
 message CommonTlsContext {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.extensions.transport_sockets.tls.v3.CommonTlsContext";
 
+  // Config for CA client plugin to get secrets
+  message CAClientPluginConfig {
+    option (udpa.annotations.versioning).previous_message_type =
+        "envoy.extensions.transport_sockets.tls.v3.CommonTlsContext.CAClientPluginConfig";
+
+    // plugin name
+    string name = 1;
+
+    // plugin specific config based on the plugin name
+    google.protobuf.Any typed_config = 3;
```

Comment:
Ditto here. Field number should be 2.

Reply:
Yes, fortunately there is tooling that automatically syncs up v4alpha files to v3 ones. So I just need to fix the v3 one.